lib/entity/domain-contacts.php

```diff
@@ -78,13 +78,17 @@ public function __construct( ?Domain_Contact $owner = null, ?Domain_Contact $adm
 		}
 	}
 
+	/**
+	 * @internal
+	 */
```

Comment:
I wonder i this one should be internal 🤔

Reply:
I thought of leaving this one marked with `@internal` because it only gets called from `Contacts::__construct`. But on a second thought, this might be useful for them I guess. Thanks, I'll change it.